test/format/wts.c

```diff
@@ -281,6 +281,9 @@ wts_open(const char *home, bool set_api, WT_CONNECTION **connp)
         CONFIG_APPEND(p, ",split_8");
     CONFIG_APPEND(p, "]");
 
+    if (g.c_verify)
+        CONFIG_APPEND(p, ",verify_metadata=true");
+
```

Comment:
I think this should just be on all the time, not conditional.

Reply:
I feel like if you don't want to verify, then you shouldn't. I think if one is doing a reopen and the HS file is huge, it could be a long time. I'm going to merge as-is. But that's as hard as I'm going to push back. If you feel strongly we can open a ticket and make this tiny change.